gems/activerecord/7.0/activerecord-7.0.rbs

```diff
@@ -1,4 +1,9 @@
 module ActiveRecord
+  class Base
+    include Encryption::EncryptableRecord
+    extend ActiveRecord::Encryption::EncryptableRecord::ClassMethods
```

Comment:
Is there any reason to use both relative reference and absolute reference?

* `include Encryption::...`
* `extend ActiveRecord::Encryption::...`

Reply:
No, there is not. Correction.

Reply:
LGTM